### DISCUSSION TITLE
Definition of actively contributing

### DISCUSSION THREAD

**Discussion by emmanuelbernard** [commonhaus/foundation#14] [Q&A]
Actively contributing is used every now and then with no definition.
What is an active contributor?
How are then determined?

They imply votership so that's important

**Comment by ebullient** [comment]
Absolutely. This is... a finger in the air for now. An idea. So much of this would need to be nailed down by that first council. 

I have some other ideas in this space: 
- The Gnome Foundation has a membership application process (which feels too heavy): https://foundation.gnome.org/membership/
- JHipster has a nominal membership fee (a confirmed opt-in to being a voting member). JHipster is incorporated in France, so that fee is €1 (or EUR 1, or however you write that)

The most relevant bits from the Gnome Foundation:
> The Membership consists of all GNOME contributors who wish to become part of the Membership and go through the application process. Members can run for election to the Board of Directors, vote in the elections for the Board of Directors, and suggest referenda. The membership process is overseen by the Membership and Elections Committee.

I would like to trim some of this process down during the Bootstrapping phase. If/when we have enough members, then a Membership committee could be a good thing.

**Comment by kenfinnigan** [comment]
OpenTelemetry, part of the CNCF, has a concept of [Members of Standing](https://github.com/open-telemetry/community/blob/main/governance-charter.md#members-of-standing) to define who is eligible to vote in elections

**Reply by ebullient** [reply]
I've also been thinking about the difference between coders, doc writers, and other members of the ecosystem that may want to contribute...

The OpenTelemetry definition is pretty sound... let me see if I can make it friendlier for those like me with the attention span of a flea that fall asleep when reading stuff like this.

**Comment by kenfinnigan** [comment]
https://github.com/commonhaus/foundation-draft/blob/main/bylaws/2-cf-membership.md#general-members defines what's required to gain membership through active participation
